jujugui/static/gui/src/app/components/env-switcher/list/list.js

```diff
@@ -40,7 +40,14 @@ YUI.add('env-list', function() {
 
     generateEnvList: function() {
       var envs = [];
-      this.state.envs.forEach(function(env) {
+      const envList = this.state.envs;
+      if (envList.length === 0) {
+        return (<li className="env-list__environment"
+          key="none">
+          No models
```

Comment:
"No models available, click below to view your profile and create a new model" 

?